### PR TITLE
refactor(rxCollapse): move components/rxCollapse to elements/Collapse

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-759

### LGTMs
- [x] Dev LGTM
- [x] Dev/Design LGTM

### CODE REVIEW

**Comment by 100stacks** [comment]
@kevincareer427, please add the new demo page and API screenshots.

**Comment by kevincareer427** [comment]
## Before
before

## After
after

**Comment by 100stacks** [comment]
@kevincareer427, I updated your PR so that we can have a `Dev` or `Design` LGTM vs a second `Dev` LGTM.

@glynnis, please let me know if this works for you, or if you prefer to make the second PR reviewer strictly `Design` LGTM. 💭  🔮

File: utils/rx-page-objects/typings/globals/angular-protractor/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,1844 @@
+// Generated by typings
+// Source: https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/f9c44651705f574f6d4258fe5e1c335462bdcc19/angular-protractor/angular-protractor.d.ts
+declare namespace protractor {
+    //region Wrapped webdriver Items
```
Comment (100stacks) [inline utils/rx-page-objects/typings/globals/angular-protractor/index.d.ts]: @kevincareer427, it looks like you may have pulled in @Droogans TypeScript updates.  Please revert.

File: utils/rx-page-objects/typings/globals/chai-as-promised/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,285 @@
+// Generated by typings
+// Source: https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/2a83c342f769534a010cf91bbaf57583098fa327/chai-as-promised/chai-as-promised.d.ts
+declare module 'chai-as-promised' {
+    function chaiAsPromised(chai: any, utils: any): void;
```
Comment (100stacks) [inline utils/rx-page-objects/typings/globals/chai-as-promised/index.d.ts]: @kevincareer427, it looks like you may have pulled in @Droogans TypeScript updates.  Please revert.

File: utils/rx-page-objects/typings/globals/chai/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,405 @@
+// Generated by typings
+// Source: https://raw.githubusercontent.com/DefinitelyTyped/DefinitelyTyped/7273c57f5b35de28b77649d9160f557906a95c68/chai/chai.d.ts
+declare namespace Chai {
+
```
Comment (100stacks) [inline utils/rx-page-objects/typings/globals/chai/index.d.ts]: @kevincareer427, it looks like you may have pulled in @Droogans TypeScript updates.  Please revert.  

Same for the other extra files. 💯

File: src/elements/Collapse/rxCollapse.meta.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+{
+    "displayName": "rxCollapse",
```
Comment (100stacks) [inline src/elements/Collapse/rxCollapse.meta.json]: I think this should display `Collapse`. 💭

**Pull request merged**